app/src/main/java/com/github/sdp/tarjetakuna/ui/webapi/WebApiFragment.kt

```diff
@@ -45,7 +45,7 @@ class WebApiFragment : Fragment() {
         binding.apiCards.setOnClickListener {
```

Comment:
should change the apiCards to apiRandomCard

Reply:
True